docs/_includes/admin-tasks-content.md

```diff
@@ -825,6 +825,19 @@ Click on ‘Templates’ in the admin menu. You will see a table of existing tem
 beside the status you wish to change. You may alter any of the fields as described in the section
 above, ‘Creating a template’. Additionally you can delete the template from this page.
 
+
+#### HTML content in templates
+
+HTML tags are permitted in response templates, which makes it possible to include
+hyperlinks or rich text formatting in the updates which are added to reports.
+
+Be aware that response templates are emailed to users as well as being shown on
+the site, so it's best to keep any HTML formatting quite light-touch due to the
+quirks of email clients' rendering of HTML message.
+
+Refer to the section ["HTML Content in notices"](#html-content-in-notices) above for details of
+what tags and attributes are allowed.
```

Comment:
As html_para is applied to updates, do we want to limit this to inline tags only? Otherwise if they use `<ul>` etc it's going to get wrapped in `<p>`s on top, unless they happen to do the HTML all on one line...

Alternative would be to not do html_para if there is HTML and then mandate they must use valid HTML including start/end `<p>` (or whatever), but then we only have styling for `item-list__update-text p` for example. 

Ideally I assume we'd want to allow list items, no reason why you wouldn't want to, in which case I'd say you'd probably want to have the staff_sanitize function do combined what it does currently, markup, and html_para, depending on whether it needed to or not - I guess if the output had no p/ul/ol in it, then run html_para, but don't if it already does?